pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -38,7 +38,7 @@ const (
 	DefaultVolumePluginDir       = "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/"
 
 	// See https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2570-memory-qos
-	DefaultMemoryThrottlingFactor = 0.8
+	DefaultMemoryThrottlingFactor = 0.9
```

Comment:
changing config file defaults is surprising... this means that identical kubelet config files that omit this value will behave differently between 1.26 and 1.27? will the different factor break any existing working users?

Reply:
oh, I just realized this is still an alpha-gated feature. adjusting while in alpha is fine